en/sql/identifier.rst

```diff
@@ -43,12 +43,28 @@ Identifiers or reserved keywords shown as below are not allowed. However, if the
 
 ::
 
-    " select"
-    " @lowcost"
-    " low cost"
-    " abc" " def"
+    "select"
+    "@lowcost"
+    "low cost"
+    "abc" "def"
     [position]
 
+.. Warning::
+
+    Even identifiers enclosed in double quotation marks, square brackets, or backticks cannot contain square brackets ([ ]) or dots (.).
```

Comment:
```suggestion
    Even though identifiers are enclosed in double quotation marks, square brackets, or brackets, they cannot contain square brackets ([ ]) or dots (.).
```
I guess in here 'backpacks' is a typo of 'brackets'? 
And also, I am not sure whether the way I corrected is the same as the original meaning of this sentence, please check again.

Reply:
backticks means " ` " sign. It's not a typo.

Reply:
Thanks, I'll change it except for backticks.